bin/generate-mailing-list-us-only.rb

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env ruby
+require_relative '../pegasus/src/env'
+require 'cdo/solr'
+require src_dir 'database'
+
+SETS = {
```

Comment:
Yeah, and the methods that aren't used here like city_query, etc

On Wed, Jul 15, 2015 at 11:28 AM, Andre Stackhouse <notifications@github.com
> wrote:

> In bin/generate-mailing-list-us-only.rb
> <https://github.com/code-dot-org/code-dot-org/pull/3123#discussion_r34712134>
> :
>
> > @@ -0,0 +1,173 @@
> > +#!/usr/bin/env ruby
> > +require_relative '../pegasus/src/env'
> > +require 'cdo/solr'
> > +require src_dir 'database'
> > +
> > +SETS = {
>
> So that would be everything except UNSUBSCRIBERS and TEACHERS?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/3123/files#r34712134>.
>